clusterDemo/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "fmt"
+import ci "scoot/clusterImplementations"
+
+func main() {
+
+  var cluster = ci.StaticLocalNodeClusterFactory(10)
```

Comment:
done! except when declaring empty slice in test files to avoid allocating unnecessary memory